ansible/roles/invoker/tasks/deploy.yml

```diff
@@ -173,8 +173,8 @@
         {% for item in (invoker_container_network_dns_servers | default()).split(' ')  %}
         -e CONFIG_whisk_containerFactory_containerArgs_dnsServers_{{loop.index0}}={{ item }}
         {% endfor %}
-        -e INVOKER_NUMCORE='{{ invoker.numcore }}'
-        -e INVOKER_CORESHARE='{{ invoker.coreshare }}'
+        -e CONFIG_whisk_containerFactory_containerArgs_numCore='{{ invoker.numcore }}'
+        -e CONFIG_whisk_containerFactory_containerArgs_coreShare='{{ invoker.coreshare }}'
```

Comment:
Hmm, should these live under container-args? Feels like they should live on `whisk.containerpool` instead?

Reply:
I'm not sure why - these are used primarily to set the `--cpushares` flag for docker run; it is from that perspective that the container pool also uses it to define "maximumContainers" (to define the pool size). 

FWIW in the future, I will be creating a PR to isolate these 2 items (cpushare vs maximumContainers) , since it does not make sense for cluster-aware deployments where maximumContainers value can fluctuate dynamically, and is not correlated to only the setting used for `--cpushares`.

Reply:
Its main purpose is to determine the maximum amount of containers to spawn. Secondarily, that value is used to also determine the cpushares parameter, but the main purpose is to tell the containerpool: How many cores and how do you want to split them.

Reply:
Ha. Ok moving it.

Reply:
Oh separately from config path, there is another issue, where ContainerPoolConfig should generally reside near ContainerPool which resides in invoker, but ContainerFactory resides in common. This is a problem because non-docker ContainerFactory will require use of this config (to set the cpushares), so for now I will use `whisk.container-factory.container-pool` (not `whisk.container-pool`) to store the config.

Reply:
nm, I can just leave it at `whisk.container-pool`, but the config class will be in ContainerFactory. This class can move into ContainerPool in the future if/when ContainerPool moves to common.